src/applicant-api.js

```diff
@@ -1,12 +1,22 @@
 const applicantApi = {
+    storage: localStorage,
     save(applicant) {
-        const json = JSON.stringify(applicant);
-        localStorage.setItem('applicant', json);
+        const applicants = applicantApi.getAll();
+        applicants.push(applicant);
+        const json = JSON.stringify(applicants);
+        applicantApi.storage.setItem('applicants', json);
     },
     get() {
```

Comment:
Looks like your get method isn'y quite done yet. It should take a parameter like 
``` get(name) {
 // do something with that name 
// Probably need to use a for loop. 
}